test/exceed_test.gd

```diff
@@ -0,0 +1,279 @@
+class_name ExceedGutTest
+extends GutTest
+
+var game_logic : LocalGame
+var next_test_card_id = 50000
+
+var player1 : LocalGame.Player
+var player2 : LocalGame.Player
+
+### !!! IMPORTANT
+# Subclasses must override this function
+func who_am_i():
+	return "no-one-lmao"
```

Comment:
maybe a descriptive string here on what you intend this to be in the overridden func

Reply:
Done.